soccer/gameplay/tactics/positions/goalie.py

```diff
@@ -164,17 +170,15 @@ def on_exit_clear(self):
         self.remove_subbehavior('kick-clear')
 
     def on_enter_intercept(self):
-        i = skills.intercept.Intercept()
+        self.isInter = True
+        i = skills.intercept.Intercept(None, False)
         self.add_subbehavior(i, 'intercept', required=True)
 
     def execute_intercept(self):
-        ball_path = robocup.Segment(
-            main.ball().pos,
-            main.ball().pos + main.ball().vel.normalized() * 10.0)
-        dest = ball_path.nearest_point(self.robot.pos)
-        self.robot.move_to(dest)
+        pass
```

Comment:
If we don't need `execute_intercept` anymore, it's probably better to just completely remove the function stub.